python/ray/serve/tests/test_proxy_state.py

```diff
@@ -294,13 +294,23 @@ def test_proxy_state_update_starting_ready_succeed():
     # Ensure the proxy status before update is STARTING.
     assert proxy_state.status == ProxyStatus.STARTING
 
+    # Ensure actor_details are set to the initial state when the proxy_state is created.
```

Comment:
These assertions are checking implementation details. Is there any way to check for the same behavior using public attributes/methods?

Reply:
like do `actor_details` instead of `_actor_details`? lol Alternatively we can mock and check `update_actor_details` is called with the correct args, but I feel that's less desired

Reply:
Oh awesome, yeah `actor_details` works.